app/Http/Controllers/Store/BundleController.php

```diff
@@ -162,10 +181,47 @@ function ($value) {
             $errors[] = $bundle->syncVouchers($voucherCodes);
         }
 
-        // TODO : disbursing bundle code to go here.
+        // Check we have values on our inputs; This should have been covered in validation...
+        if (isset($inputs['collected_at']) &&
+            isset($inputs['collected_by']) &&
+            isset($inputs['collected_on'])
+        ) {
+            // Add the date;
+            $bundle->disbursed_at = Carbon::createFromFormat(
+                'Y-m-d',
+                $inputs['collected_on']
```

Comment:
Disbursed is the same as collected? I presume that makes sense from the data's pov but it's a bit confusing to read.